src/main/java/study/StringAddCalculator.java

```diff
@@ -0,0 +1,50 @@
+package study;
+
+import java.util.Arrays;
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+
+public class StringAddCalculator {
+
+    private static final Pattern pattern = Pattern.compile("//(.)\n(.*)");
+    private static final String DEFAULT_SPLITER = "[,:]";
+
+    public static int splitAndSum(String value) {
+        if (value == null || value.isEmpty()) {
+            return 0;
+        }
+
+        var spliter = getSpliter(value);
+        var toBeSplitValue = getToBeSplitValue(value);
+
+        var tokens = toBeSplitValue.split(spliter);
+
+        if (hasNegativeNumber(tokens)) {
+            throw new RuntimeException();
```

Comment:
```suggestion
            throw new IllegalArgumentException("적절한 예외 메시지");
```
RuntimeException보다는 상황에 맞는 예외를 사용하는 게 좋아요
예제에서는 표준 예외를 넣어봤지만 상황에 맞는 커스텀 예외를 만들어 사용하는 것도 좋은 방법입니다.

그리고 예외를 잡아서 처리하는 입장에서는 왜 예외가 발생했는지 알기 어려우면 디버깅이 힘들어질 수 있습니다.
그래서 예외를 던질 때는 의미 있는 메시지를 같이 넣어주는 걸 추천합니다!